uiowa/uiowa
# Added file field to thesis defense

# How to test


1. blt ds --site=genetics.grad.uiowa.edu (since grad.uiowa.edu is having db issues rn)
2. make sure can add file field to thesis defense content type
3. boom
